README.md

```diff
@@ -13,7 +13,7 @@ It leverages the [Membrane RTC Engine](https://github.com/jellyfish-dev/membrane
 
 There are two ways of running Jellyfish:
 - building from source (requires Elixir and native dependencies)
-- using Jellyfish Docker images
+- using Jellyfish Docker images (Remember that you should set `INTEGRATED_TURN_IP` env to your public IP address and `INTEGRATED_TURN_LISTEN_IP` to `0.0.0.0`).
```

Comment:
Could we move this to jellyfish docs?  https://jellyfish-dev.github.io/jellyfish-docs/getting_started/peers/webrtc

Reply:
I think it could stay here, but I also added it to the jellyfish docs.
https://github.com/jellyfish-dev/jellyfish-docs/pull/12